src/components/search/start-travel.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { View, Text, StyleSheet, Platform, Dimensions } from "react-native";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import { Entypo } from "@expo/vector-icons";
+import { CONCORDIA_RED } from "../../constants/style";
+
+interface IProps {
+  setStartTravelPlan: (bool: Boolean) => void;
+}
+
+/**
+ * Component for controlling indoor floors when in the indoor view.
+ *
+ * @param indoorInformation Indoor information object
+ * @param onFloorPickerButtonPress Function that handles floorpicker press event
+ */
+const StartTravel = ({ setStartTravelPlan }: IProps) => {
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        onPress={() => setStartTravelPlan(true)}
+        style={{ width: 80, height: 80 }}
```

Comment:
No styling in the jsx please. Also the button is too close to the right side and too big (on android at least).